TS formatting: workspaces & layouts

duianto opened the pull request:
```
Remove empty lines:
- Two in the Layouts Transients State.
- Three in the Workspaces Transient State.

In the Layouts Transient State:

Move the navigation keys:
[n/C-l]    next layout
[N/p/C-h]  prev layout
closer to the top, to match the order in the Workspaces Transient State.

Change to a more generic description, without mentioning the other
completion name.
From: [l] layout w/helm/ivy
To:   [l] another layout

Shorten the descriptions:
From: [w] workspaces transient state
To:   [w] workspaces TS
and
From: [t] show a buffer without adding it to current layout
To:   [t] show buffer w/o adding to layout

Indicate what's being added:
From: [A] add all from layout
To:   [A] add all buffers from layout

In the Workspaces Transient State:

Rename "create" to "clone, because it's more descriptive and the
docstring for the variable "eyebrowse-new-workspace" says:
"nil: Clone last workspace."
From: [c/C] create workspace
To:   [c/C] clone workspace

Add "TS" to the description to indicate that it switches to the Layouts
Transient State and then it will match "workspaces TS" in the Layouts
Transient State.
From: [l] layouts
To:   [l] layouts TS

Instead of sorting "[l] layouts TS" alphabetically by it's key letter,
move it to the bottom of the "Go to" column. This has two benefits:
- The workspaces navigation keys are grouped together, without having
"[l] layouts TS" in the middle.
- And "[l] layouts TS" will be the last entry in the "Go to" column.
This matches the location of "[w] workspaces TS" in the Layouts
Transient State.

Change to a more generic description, without mentioning the other
completion name.
From: [w] workspace w/helm/ivy
To:   [w] another workspace
```
---

Layouts Transient State, Before and After

![layouts](https://user-images.githubusercontent.com/13420573/64693674-15f4e800-d498-11e9-8579-62577d701d58.png)

Workspaces Transient State, Before and After

![workspaces](https://user-images.githubusercontent.com/13420573/64693689-1ee5b980-d498-11e9-818e-ab9f98d1d1e7.png)

smile13241324 commented:
@duianto I think I have merged a similiar PR from you today. Would you mind double checking that this PR is still necessary and fix the conflicts?

duianto commented:
I fixed the conflicts, updated the commit message and screenshots.

This PR reorders some keys and rewrites some descriptions, etc.

The other PR added a two new key bindings, moved two keys and removed a single line break.
Workspaces TS: Add single win workspace key #12317

smile13241324 commented:
Merged
